api/export-doc.sh

```diff
@@ -15,15 +15,18 @@ cp v1/README.md $TMP_FILE
 
 # cleanup the generated documentation for the website
 # remove the link to the top of the page
-sed -i '/<p align="right"><a href="#top">Top<\/a><\/p>/d' $TMP_FILE
-sed -i '/<a name="top"><\/a>/d' $TMP_FILE
+sed -i'' -e '/<p align="right"><a href="#top">Top<\/a><\/p>/d' $TMP_FILE
+sed -i'' -e '/<a name="top"><\/a>/d' $TMP_FILE
 # remove title
-sed -i '/^# Protocol Documentation/d' $TMP_FILE
+sed -i'' -e '/^# Protocol Documentation/d' $TMP_FILE
 # remove table of content
-sed -i -e '/## Table of Contents/,/##/{//!d}' $TMP_FILE
-sed -i '/^## Table of Contents/d' $TMP_FILE
+sed -i'' -e '/## Table of Contents/,/##/{//!d}' $TMP_FILE
+sed -i'' -e '/^## Table of Contents/d' $TMP_FILE
 # cleanup unecessary consecutive whitelines
-sed -i '/^[[:space:]]*$/N;/^[[:space:]]*\n[[:space:]]*$/D' $TMP_FILE
+sed -i'' -e '/^[[:space:]]*$/N;/^[[:space:]]*\n[[:space:]]*$/D' $TMP_FILE
+
+# Cleanup backup file that might be created by macOS sed:
+rm ${TMP_FILE}-e || true
```

Comment:
So I did not experience the backup file that might be created, if that happens, are you sure about its name (the `-e` suffix)?

Anyway could you suppress the error from rm `rm: cannot remove '/tmp/tmp.V2HaNn1uox-e': No such file or directory` because it looks something wrong happened

```suggestion
rm ${TMP_FILE}-e 2>/dev/null || true
```

Reply:
> So I did not experience the backup file that might be created

On macOS or Linux? I've tested on macOS and did not get this `rm: cannot remove` error. Whatever, I've added the `2>/dev/null` as I think it makes sense either way. Thanks!